src/fsharp/fsi/fsi.fsproj

```diff
@@ -33,9 +33,6 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\FSharp.Compiler.Private\FSharp.Compiler.Private.fsproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\FSharp.Compiler.Interactive.Settings\FSharp.Compiler.Interactive.Settings.fsproj" />
 
-    <!-- only used when '$(TargetFramework)' == 'netstandard2.0' -->
-    <ProjectReference Include="..\FSharp.Build\FSharp.Build.fsproj" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
```

Comment:
This was a merge error that hasn't been corrected for `release/dev16.2` but came along for the ride during regular branch merges.  `$(TargetFramework)` is never `netstandard2.0` so it's a noop anyways.